Source/Modules/javascript.cxx

```diff
@@ -2719,6 +2720,15 @@ int TypeScriptTypes::classHandler(Node *n)
   int returnValue = Language::classHandler(n);
   if (generateTsTypes)
   {
+    if (!emittedImport) {
+      String *swigType = NewString("SWIGTYPE");
```

Comment:
This is a bit suspicious, we're supposed to pass a `SwigType`, not `String`, to `typemapLookup()`. If it works, all the better, but I just wanted to check.

Reply:
SwigType is basically a string. If you look at NewSwigType(int t), it receives an intege (used as enumerated) and return a NewString() . I tried to use SwigType_default_create() but couldn't have it working :-(, so I used with a string and it worked. I know it is suspicious and I hate it....but it worked :-(